### ISSUE TITLE
Dependency to Spring boot 2.2.0.M3

### ISSUE THREAD

**Issue by mmaeller** [freefair/okhttp-spring-boot#14]
```java
Failed to collect dependencies at io.freefair.okhttp-spring-boot:okhttp3-spring-boot-starter:jar:2.1.5.2: Failed to read artifact descriptor for io.freefair.okhttp-spring-boot:okhttp3-spring-boot-starter:jar:2.1.5.2: Could not find artifact org.springframework.boot:spring-boot-dependencies:pom:2.2.0.M3 in maven central (http://central.maven.org/maven2/)
```
Since version 2.1.5.1 the spring milestone repository seems to be required.
2.1.5 works fine.
Could you please take a look?

Kind regards,
Matthias

**Comment by larsgrefer** [comment]
Oops 😅 

Just downgrade to 2.1.5 in the meantime

**Comment by larsgrefer** [comment]
Should be fixed with 2.1.6 now

**Comment by mmaeller** [comment]
Works, thanks 👍🏻
